src/index.html

```diff
@@ -8,6 +8,107 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <span class="stars__star">
+
+      </span>
```

Comment:
just this way, much simpler
```suggestion
      <span class="stars__star"></span>
```